plmfit/models/lightning_model.py

```diff
@@ -268,7 +268,6 @@ def on_validation_epoch_end(self):
             self.trainer.save_checkpoint(f'{self.plmfit_logger.base_dir}/lightning_logs/best_model.ckpt')
             self.best_epoch = self.current_epoch
             self.epochs_no_improve = 0
-            #self.plmfit_logger.log(f'Best model saved at epoch {self.best_epoch}')
         else:
             self.epochs_no_improve += 1
         self.plmfit_logger.log(f'Best model saved at epoch {self.best_epoch}')
```

Comment:
Does this make sense? Shouldn't it log the best model only if a new best checkpoint is saved or when the early stopping is activated?